### PR TITLE
Add multiple camera support.

### PR DESCRIPTION
**Pull request by michael-chi**
1. Allow the user to generate multiple camera application yaml files so that they can install multiple camera application pods on the Anthos cluster.
2. Allow the user to generate application yaml files without rebuilding container images and regenerate secrets.
3. Add KUBECONFIG to ${home}/.bashrc

### CODE REVIEW

File: edge-server/anthos/deploy-app.sh [hunk 1]
```diff
@@ -125,5 +125,11 @@ apply_kubernetes_menifest "${KUBE_MENIFEST_FOLDER}/namespace.yaml" "${KUBECONFIG
 apply_kubernetes_menifest "${KUBE_MENIFEST_FOLDER}/secret_image_pull.yaml" "${KUBECONFIG_PATH}"
 apply_kubernetes_menifest "${KUBE_MENIFEST_FOLDER}/secret_pubsub.yaml" "${KUBECONFIG_PATH}"
 apply_kubernetes_menifest "${KUBE_MENIFEST_FOLDER}/mosquitto.yaml" "${KUBECONFIG_PATH}"
-apply_kubernetes_menifest "${KUBE_MENIFEST_FOLDER}/viai-camera-integration.yaml" "${KUBECONFIG_PATH}"
+
+# It's iterate through files in a directory, not string elements in an array
```
Comment (ferrarimarco) [inline edge-server/anthos/deploy-app.sh:129]: Remove `It's`

File: edge-server/anthos/node-setup-common.sh.tmpl [hunk 2]
```diff
@@ -225,3 +225,8 @@ setup_nvidia_container_runtime() {
   fi
 
 }
+
+export_kubeconfig_path() {
```
Comment (ferrarimarco) [inline edge-server/anthos/node-setup-common.sh.tmpl:229]: Why do you need these variables there? Do all users need these variables defined? What if the user doesn't use bash?

See [this page](https://blog.flowblok.id.au/2013-02/shell-startup-scripts.html) for more info about where to put these variables according to your use case.

File: scripts/0-generate-viai-application-assets.sh [hunk 3]
```diff
@@ -187,108 +209,143 @@ VIAI_CAMERA_INTEGRATION_DIRECTORY_PATH=$DEPLOYMENT_TEMP_FOLDER
...
+###################################################
+CAMERA_ID_START=
+CAMERA_ID_END=
+
+if [ -n "${CAMERA_ID_RANGE}" ]; then
+  IFS='-'
+  # We need to original value, so do not double quote the string. Otherwise we are unable to split it into an array
   # shellcheck disable=SC2086
-  exit $EXIT_GENERIC_ERR
+  set -- $CAMERA_ID_RANGE""
```
Comment (ferrarimarco) [inline scripts/0-generate-viai-application-assets.sh:279]: Maybe missing an `=`?

**Review by ferrarimarco — CHANGES_REQUESTED** [review]
A few things to check. For the next changes, I suggest splitting the PR in smaller PRs that are easier to review :).

File: scripts/0-generate-viai-application-assets.sh (hunk 3 above)
Comment (michael-chi) [reply scripts/0-generate-viai-application-assets.sh]: fixed, removed those double quotes.

File: edge-server/anthos/node-setup-common.sh.tmpl (hunk 2 above)
Comment (michael-chi) [reply edge-server/anthos/node-setup-common.sh.tmpl]: remove this for now. will invetisgate further and resumit another PR for this.
this is based on discussion with Markku to let users easier to collect images..etc.

File: edge-server/anthos/deploy-app.sh (hunk 1 above)
Comment (michael-chi) [reply edge-server/anthos/deploy-app.sh]: fixed

File: readme.md [hunk 4]
```diff
@@ -37,9 +37,12 @@ scripts/provisioning-terraform.sh \
...
 ```
 
-## Anthos Setup
+## Edge Server Setup
 
-### Create `anthos` VIAI application assets use `Artifacts Registry` and `Cloud Build`, and create a `.ISO` file for Anthos Bare Metal set up
+### Create VIAI application assets with one camera
+
+* Use `Cloud Build` to build the camera application container image
+* Store the container image in `Artifacts Registry`
```
Comment (ferrarimarco) [inline readme.md:45]: `Artifact Registry`

**Review by ferrarimarco — DISMISSED** [review]

File: readme.md (hunk 4 above)
Comment (michael-chi) [reply readme.md:45]: fixed

**Review by ferrarimarco — DISMISSED** [review]

File: kubernetes/viai-model/viai-model.yaml.tmpl [hunk 5]
```diff
@@ -21,6 +21,11 @@ spec:
         - name: regcred
       containers:
       - image: viai-inference-module
+        resources:
+          requests:
+            nvidia.com/gpu: 1
```
Comment (ferrarimarco) [inline kubernetes/viai-model/viai-model.yaml.tmpl:26]: Is this (and the following limit) expected to be hard coded to `1`?

**Review by ferrarimarco — APPROVED** [review]

File: kubernetes/viai-model/viai-model.yaml.tmpl (hunk 5 above)
Comment (michael-chi) [reply kubernetes/viai-model/viai-model.yaml.tmpl:26]: yes, customer can customize it to other values, for example, `0` if they don't run a GPU model. It's up to the customer to decide.
I have a customization section in the document, will mention this in the document as well.

**Pull request merged**